third_party/amd/lib/TritonAMDGPUToLLVM/Utility.cpp

```diff
@@ -693,4 +693,50 @@ bool isChainDotTail(tt::DotOpInterface dotOp) {
   return false;
 }
 
+AliasScopeDomainAttr getLoadScopeDomain(MLIRContext *ctx) {
+  return AliasScopeDomainAttr::get(
+      ctx, StringAttr::get(ctx, "AsyncOps"),
```

Comment:
Prefix these domain/scope names with `amdgpu.` to avoid potential name collision?